data/nlu/nlu.md

```diff
@@ -3125,6 +3127,7 @@
 - wordpress
 - slack
 - e commerce bot
+- cicciopasticcio
```

Comment:
should some of these be annotated with [company]? maybe that is already in master as well, not sure.